assets/src/block-validation/sidebar.js

```diff
@@ -74,113 +49,37 @@ export function Sidebar() {
-								<ExternalLink href={ reviewLink } className="amp-sidebar__review-link">
-									{ __( 'View technical details', 'amp' ) }
-								</ExternalLink>
-							) }
-						</p>
-					</div>
-					{ hasReviewedValidationErrors && (
-						<div className="amp-sidebar__options">
-							<ToggleControl
-								checked={ isShowingReviewed }
-								label={ __( 'Include reviewed issues', 'amp' ) }
-								onChange={ ( newIsShowingReviewed ) => {
-									setIsShowingReviewed( newIsShowingReviewed );
-								} }
-							/>
-						</div>
-					) }
-				</PanelBody>
-			) }
+			<SidebarNotificationsContainer isShady={ true }>
+				<AMPRevalidateNotification />
+				<AMPValidationStatusNotification />
+			</SidebarNotificationsContainer>
 
-			{ isFetchingErrors ? (
-				<PanelBody opened={ true }>
-					<Loading />
-				</PanelBody>
-			) : (
-				<>
-					{ ! isPostDirty && validationErrors.length === 0 && (
-						<PanelBody opened={ true }>
-							<PanelRow>
-								<p>
-									{ isEditedPostNew
-										? __( 'Validation issues will be checked for when the post is saved.', 'amp' )
-										: __( 'There are no AMP validation issues.', 'amp' ) }
-								</p>
-							</PanelRow>
-						</PanelBody>
-					) }
-
-					{ isPostDirty && (
-						<PanelBody opened={ true }>
-							<PanelRow>
-								<p>
-									{ isEditedPostNew
-										? __( 'Validation issues will be checked for when the post is saved.', 'amp' )
-										: __( 'The post content has been modified since the last AMP validation.', 'amp' ) }
-								</p>
-							</PanelRow>
-							<PanelRow>
-								{ isDraft ? (
-									<Button isSecondary onClick={ () => savePost( { isPreview: true } ) }>
-										{ __( 'Save draft and validate now', 'amp' ) }
-									</Button>
-								) : (
-									<Button isSecondary onClick={ () => autosave( { isPreview: true } ) }>
-										{ __( 'Re-validate now', 'amp' ) }
-									</Button>
-								) }
-							</PanelRow>
-						</PanelBody>
-					) }
-				</>
+			{ 0 < displayedErrors.length && (
+				<ul className="amp-sidebar__errors-list">
+					{ displayedErrors.map( ( validationError, index ) => (
+						<li
+							// Add `index` to key since not all errors have `clientId`.
+							key={ `${ validationError.clientId }${ index }` }
+							className="amp-sidebar__errors-list-item"
+						>
+							<Error { ...validationError } />
+						</li>
+					) ) }
+				</ul>
 			) }
 
-			{ 0 < validationErrors.length && (
-				0 < displayedErrors.length ? (
-					<ul>
-						{ displayedErrors.map( ( validationError, index ) => (
-							<Error { ...validationError } key={ `${ validationError.clientId }${ index }` } />
-						) ) }
-					</ul>
-				)
-					: ! isDraft && (
-						<PanelBody opened={ true }>
-							<p>
-								{ __( 'All AMP validation issues have been reviewed.', 'amp' ) }
-							</p>
-						</PanelBody>
-					)
+			{ hasReviewedValidationErrors && (
+				<div className="amp-sidebar__options">
```

Comment:
The idea is that this would be clicked infrequently. The main thing a user will care about are the unreviewed issues. The link to show all is there just for curiosity.